From: SeongJae Park <sj@kernel.org>
Date: Fri, 3 Oct 2025 12:59:35 -0700
Subject: [PATCH] ==== test_ctx alloc fix ====

mm/damon/sysfs: fix commit test damon_ctx [de]allocation

DAMON sysfs interface dynamically allocates and uses a damon_ctx object
for testing if given inputs for online DAMON parameters update is valid.
The object is being used without an allocation failure check, and leaked
when the test succeeds.  Fix the two bugs.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_epjb9n3u | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_epjb9n3u

diff --git a/hkml_cv_bogus/hkml_cv_bogus_epjb9n3u b/hkml_cv_bogus/hkml_cv_bogus_epjb9n3u
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

